app/models/topic.rb

```diff
@@ -30,6 +30,19 @@ def self.max_fancy_title_length
     400
   end
 
+  LARGE_TOPIC_POSTS_COUNT = 500
+  MEGA_TOPIC_POSTS_COUNT = 10000
+
+  # A 'large' topic has certain meta-information cached.
+  def self.large_topic_posts_count
+    LARGE_TOPIC_POSTS_COUNT
```

Comment:
What's your reasoning for having a CONSTANT as well as a `self.` method? Usually we do one or the other.